src/app.rs

```diff
@@ -271,6 +272,16 @@ where
             false
         }
     }
+
+    /// Runs the app inside a window.
+    pub fn run_with_title(self, title: impl Into<String>) {
+        AppLauncher::new(self).title(title).run()
+    }
+
+    /// Runs the app inside a window.
```

Comment:
The documentation here should mention that this is a convenience around `AppLauncher`, and point the user in the direction of that struct when they need to do something more complex.

Same for `run_with_title`.

Reply:
I agree. I think this would make the code more engaging and rewarding to discover for oneself.

Reply:
It is addressed in the latest commit